pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumloadbalancerippools.yaml

```diff
@@ -0,0 +1,214 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: ciliumloadbalancerippools.cilium.io
```

Comment:
First commit should rename `CiliumBGPLoadBalancerIPPool` into `CiliumLoadBalancerIPPool`, but I note that there are some other changes in the commit (the `_types.go` and YAML files have been expanded for the new CRD, and it no longer depends on BGP in `pkg/k8s/synced/crd.go`). Would be nice to mention that in the commit description.